packages/api/README.md

```diff
@@ -160,6 +160,87 @@ console.log(rt3.length) // => 25
 console.log(rt3.graphemeLength) // => 1
 ```
 
+### Moderation
```

Comment:
is there an argument for putting this in a separate package?

we use `@atproto/api` in a the pds/appview for instance and may not want to pull in the whole labeling sdk

don't think this is block but could imagine an `sdk` package that is comprised of `api` & `labels` or similar

Reply:
Maybe but I'm not inclined to do it yet